app.js

```diff
@@ -0,0 +1,302 @@
+}
+function handleDwnClick() {
+  var indexA = event.target.parentElement.parentElement.getAttribute('roll-index');
+  indexA = parseInt(indexA);
+  swapRolls(indexA, indexA + 1);
+  renderAttributesTable();
+}
+function swapRolls(indexA, indexB) {
+  var tmp = rolls[indexA];
+  rolls[indexA] = rolls[indexB];
+  rolls[indexB] = tmp;
+}
+function getHeightFeet() {
+  var feet = document.getElementById('feet').value;
+  return feet;
+}
+function getHeightInches() {
+  var inches = document.getElementById('inches').value;
+  return inches;
+}
+function getWeight() {
+  var weight = document.getElementById('weight').value;
+  return weight;
+}
+function getStory() {
+  var story = document.getElementById('characterStory').value;
+  return story;
+}
+function getLooks() {
+  var looks = document.getElementById('looks').value;
+  return looks;
+}
+function getClass(){
+  var a = document.getElementById('class');
+  var getClass = a.options[a.selectedIndex].value;
+  return getClass;
+}
+function getRace(){
+  var a = document.getElementById('race');
+  var getRace = a.options[a.selectedIndex].value;
+  return getRace;
+}
+function getGender(){
+  var a = document.getElementById('gender');
+  var getGender = a.options[a.selectedIndex].value;
+  return getGender;
+}
+function getAlignment(){
+  var a = document.getElementById('alignment');
+  var getAlignment = a.options[a.selectedIndex].value;
+  return getAlignment;
+}
+function raceAttributes(race){
+  switch(race){
+  case 'gnome':
+    character.constitution += 2;
+    character.charisma += 2;
+    character.strength -= 2;
+    character.size = 'small';
+    break;
+  case 'human':
+    character.size = 'medium';
+    break;
+  case 'orc':
+    character.strength += 4;
+    character.intelligence -= 2;
+    character.wisdom -= 2;
+    character.charisma -= 2;
+    character.size = 'meduim';
+    break;
+  case 'elf':
+    character.dexterity += 2;
+    character.charisma += 2;
+    character.constitution -= 2;
+    character.size = 'medium';
+    break;
+  case 'halfling':
+    character.dexterity += 2;
+    character.charisma += 2;
+    character.strength -= 2;
+    character.size = 'small';
+    break;
+  case 'dwarf':
+    character.constitution += 2;
+    character.charisma -=2;
+    character.size = 'medium';
+  }
+}
+for (var i = 0; i < myCharacters.length; i++){
+  var sidebar = document.getElementById('sidebar');
+  var aTag = document.createElement('a');
+  aTag.setAttribute('href','display.html');
+  aTag.setAttribute('name', 'click');
+  aTag.setAttribute('class', 'characters');
+  // aTag.setAttribute('style', 'padding: 20px')
+  aTag.setAttribute('id', i);
+  aTag.innerHTML = myCharacters[i].name;
+  sidebar.appendChild(aTag);
+}
+
+var aTags = [];
+function eventAdder (){
+  aTags = document.getElementsByName('click');
+  for (var i=0;i<aTags.length;i++){
+    var a = aTags[i];
+    a.addEventListener('click', handleSubmitClickSideBar);
+  }
+}
+eventAdder();
+
```

Comment:
great job naming your functions and variables. they are all following best practice for case, and are very thoughfull!